reporter/src/funTest/kotlin/Extensions.kt

```diff
@@ -27,3 +27,9 @@ fun String.patchAsciiDocTemplateResult() =
         // Asciidoctor renders time zones differently depending on the platform.
         // For macOS the time is rendered as `00:00:00 +0000` while for Linux it is `00:00:00 UTC`.
         .replace("""[+-]\d{4}""".toRegex(), "UTC")
+
```

Comment:
Strictly speaking, this commit does not yet "Override the UUID"; it just introduces a function to do so in the upcoming commit. I'd be fine with simply squashing this to the next commit.